pkg/log/locallogger/locallogger_test.go

```diff
@@ -73,3 +75,46 @@ func TestKitLogging(t *testing.T) {
 	}
 
 }
+
+func TestCleanUpRenamedDebugLogs(t *testing.T) {
+	t.Parallel()
+
+	tempDir := t.TempDir()
+
+	// Make a log file that should be cleaned up
+	legacyDebugLogPath := filepath.Join(tempDir, "debug.log")
+	f1, err := os.Create(legacyDebugLogPath)
+	require.NoError(t, err, "could not create log file for test")
+	defer f1.Close()
+
+	// Make a rotated log file that should be cleaned up
+	legacyDebugLogRotatedPath := filepath.Join(tempDir, "debug-2022-11-18T18-35-48.858.log.gz")
+	f2, err := os.Create(legacyDebugLogRotatedPath)
+	require.NoError(t, err, "could not create log file for test")
+	defer f2.Close()
```

Comment:
As a small nit, you don't need to defer these -- nothing is being written to them. They can get closed immediately. 

Could probably even do it in for loop  with all the names, if you wanted to be fancy WRT variables.

Reply:
Thanks for the suggestion -- tidied this up 🙂